tests/test_actor.py

```diff
@@ -33,6 +35,7 @@ def test_calls_perform_as() -> None:
     action.perform_as.assert_called_once_with(actor)
 
     perform_aliases: list[Any] = [
+        actor,
```

Comment:
It was awesome to be able to just add this here, nice test @bandophahita!

Reply:
it's because I'm so lazy.